packages/main/src/backend/configManager/configManager.ts

```diff
@@ -11,13 +12,18 @@ export async function getConfig(
   if (configFromFile) {
     const decrypted = (await decrypt(configFromFile)) as string;
     if (!decrypted) {
-      console.log('No config file found, returning default config');
+      logger.log('No config file found, returning default config');
       return configExample;
     }
     try {
-      return JSON.parse(decrypted);
+      const config = JSON.parse(decrypted);
+      if (Object.keys(config).length === 0) {
+        logger.log('Empty config file found, returning default config');
+        return configExample;
+      }
+      return config;
```

Comment:
This is a fix for my case, when the config somehow became `{}`. (Locally on dev env)